pandas/tests/indexing/test_coercion.py

```diff
@@ -928,7 +929,7 @@ class TestReplaceSeriesCoercion(CoercionBase):
     klasses = ["series"]
     method = "replace"
 
-    rep = {}
+    rep = {}  # type: Dict[str, List[Any]]
```

Comment:
Minor nit but can you just do `List` instead of `List[Any]`? Equivalent but I think the former is used throughout code base